config/locales/jobseekers.yml

```diff
@@ -237,6 +237,23 @@ en:
         phone_number: Phone number
         relationship: Relationship to applicant
       review:
+        confirmation:
+          checkbox_label:
+            confirm_data_accurate: I confirm that the above information is accurate and complete
+            confirm_data_usage: I consent to my data being shared
+          heading: Confirmation
+          how_we_use_your_data:
+            description: "When you submit your application, your data will be shared with:"
+            heading: How we use your data
+            list_html: |
+              <ul class="govuk-list govuk-list--bullet">
+                <li>the Department for Education</li>
+                <li>the school, trust or local authority which posted the job listing</li>
+                <li>the school or schools the job is at</li>
+              </ul>
```

Comment:
It makes me uncomfortable to write HTML directly in this file - I think localizations should focus on text, and views should handle markup.

We're not reusing this text anywhere else, so I suggest just putting it directly into the slim view.

Reply:
yes seeing this in the locale file makes me shudder

Reply:
There's a few more examples in en.yml and publishers.yml if you fancy wincing some more.